easybuild/easyconfigs/t/TensorFlow/TensorFlow-1.11.0-foss-2018b-Python-3.6.6.eb

```diff
@@ -0,0 +1,92 @@
+]
+
+dependencies = [
+    ('Python', '3.6.6'),
+    ('protobuf-python', '3.6.0', versionsuffix),
+]
+
+use_pip = True
+
+exts_list = [
+    ('absl-py', '0.5.0', {
+        'source_urls': ['https://pypi.python.org/packages/source/a/absl-py'],
+        'checksums': ['6fcc3c04dc881fd93d793674a42ee8c73155570eda8f8b90c4477c8522478b7b'],
+        'modulename': 'absl',
+    }),
+    ('astor', '0.7.1', {
+        'source_urls': ['https://pypi.python.org/packages/source/a/astor'],
+        'checksums': ['95c30d87a6c2cf89aa628b87398466840f0ad8652f88eb173125a6df8533fb8d'],
+    }),
+    ('gast', '0.2.0', {
+        'source_urls': ['https://pypi.python.org/packages/source/g/gast'],
+        'checksums': ['7068908321ecd2774f145193c4b34a11305bd104b4551b09273dfd1d6a374930'],
+    }),
+    ('grpcio', '1.15.0', {
+        'source_urls': ['https://pypi.python.org/packages/source/g/grpcio'],
+        'checksums': ['15a5fc23ba2e5f30758fbb8763f79a764e3de1ffcb92c2ccd46f45ffa63910d2'],
+        'modulename': 'grpc',
+    }),
+    ('Markdown', '3.0.1', {
+        'source_urls': ['https://pypi.python.org/packages/source/M/Markdown'],
+        'checksums': ['d02e0f9b04c500cde6637c11ad7c72671f359b87b9fe924b2383649d8841db7c'],
+    }),
+    ('tensorboard', '1.10.0', {
+        'source_urls': ['https://pypi.python.org/packages/source/t/tensorboard'],
+        'source_tmpl': 'tensorboard-%(version)s-py3-none-any.whl',
+        'unpack_sources': False,
+        'checksums': ['e4ea6ac2e47bf715b915f08a186e6205fa097318bd73f0b265d437b1d7834484'],
+    }),
+    ('termcolor', '1.1.0', {
+        'source_urls': ['https://pypi.python.org/packages/source/t/termcolor'],
+        'checksums': ['1d6d69ce66211143803fbc56652b41d73b4a400a2891d7bf7a1cdf4c02de613b'],
+    }),
+    ('Werkzeug', '0.14.1', {
+        'source_urls': ['https://pypi.python.org/packages/source/W/Werkzeug'],
+        'checksums': ['c3fd7a7d41976d9f44db327260e263132466836cef6f91512889ed60ad26557c'],
+    }),
+    ('Keras-Applications', '1.0.6', {
+        'source_urls': ['https://pypi.python.org/packages/source/K/Keras-Applications'],
+        'source_tmpl': 'Keras_Applications-%(version)s.tar.gz',
+        'checksums': ['a03af60ddc9c5afdae4d5c9a8dd4ca857550e0b793733a5072e0725829b87017'],
+        'modulename': 'keras_applications',
+    }),
+    ('Keras-Preprocessing', '1.0.5', {
+        'source_urls': ['https://pypi.python.org/packages/source/K/Keras-Preprocessing'],
+        'source_tmpl': 'Keras_Preprocessing-%(version)s.tar.gz',
+        'checksums': ['ef2e482c4336fcf7180244d06f4374939099daa3183816e82aee7755af35b754'],
+        'modulename': 'keras_preprocessing',
+    }),
+    (name, version, {
+        'source_urls': ['https://github.com/tensorflow/tensorflow/archive/'],
+        'source_tmpl': 'v%(version)s.tar.gz',
+        'patches': [
+            'TensorFlow-1.11.0_swig-env.patch',
+            'TensorFlow-1.11.0_remove-msse-hardcoding.patch',
+            'TensorFlow-1.11.0_fix-ToString.patch',
```

Comment:
are these patches to 1.11.0 missing from this PR or was the idea to reuse the patches for 1.10.0?